python/thunder/rdds/series.py

```diff
@@ -43,7 +43,9 @@ class Series(Data):
 
     def __init__(self, rdd, index=None, dims=None, dtype=None):
         super(Series, self).__init__(rdd, dtype=dtype)
-        self._index = index
+        self._index = None
+        if not index is None:
```

Comment:
Probably replace with `if index is not None:` as it's less ambigious, see http://stackoverflow.com/questions/2710940/python-if-x-is-not-none-or-if-not-x-is-none for discussion